spyder/app/tests/test_mainwindow.py

```diff
@@ -2117,9 +2117,11 @@ def test_help_opens_when_show_tutorial_full(main_window, qtbot):
     'Show tutorial' opens the help plugin if closed.
     """
     HELP_STR = "Help"
-    from spyder.plugins.help.plugin import HelpActions
-    help_pane_menuitem = main_window.help.get_action(
-        HelpActions.ShowSpyderTutorialAction)
+    help_pane_menuitem = None
```

Comment:
How does this work with `localization` `_("")`?

Reply:
I think this test was thought to work using the default language to run (I guess English?)

Reply:
🤦🏻‍♂️missed that this was test file.